src/main/java/edu/cornell/kfs/fp/CuFPKeyConstants.java

```diff
@@ -29,4 +29,5 @@
     public static final String REPORT_CREATE_ACCOUNTING_DOCUMENT_SHARED_DOCUMENT_TYPE = "report.create.accounting.document.shared.document.type";
     public static final String REPORT_CREATE_ACCOUNTING_DOCUMENT_SHARED_DOCUMENT_DESCRIPTION = "report.create.accounting.document.shared.document.description";
     public static final String REPORT_CREATE_ACCOUNTING_DOCUMENT_SHARED_DOCUMENT_EXPLANATION = "report.create.accounting.document.shared.document.explanation";
+    public static final String REPORT_CREATE_ACCOUNTING_DOCUMENT_XML_PROCESING_ERROR = "report.create.accounting.document.xml.processing.error";
```

Comment:
Small typo in the name of this constant; should be "...PROCESSING..." instead of "...PROCESING..."